abyss.py

```diff
@@ -138,4 +139,7 @@ def term(self):
             pass
 
 def PLUGIN_ENTRY():
-    return abyss_plugin_t()
\ No newline at end of file
+    return abyss_plugin_t()
+
+if __name__ == "__main__":

```

Comment:
I think line 144ff should be removed because running abyss.py outside the context of IDA won't work because of unknown imports

Reply:
It is not for running abyss.py outside of IDA, it is for running it as an IDA script (instead of an IDA plugin). But since you mentioned running it as a script is bad anyway, I will remove that